### DISCUSSION TITLE
Why disable `xxhash` for `rsync`?

### DISCUSSION THREAD

**Discussion by shvchk** [termux/termux-packages#19024] [Q&A] [answered]
https://github.com/termux/termux-packages/blob/aaf1024b6e0e6078353e46b131c571d649e31995/packages/rsync/build.sh#L19

Maybe @sylirre knows, since it was his commit.

**Comment by sylirre** [comment, marked as answer]
It was disabled due to compilation issues.

In general it's a common practice for us to disable stuff when it cause ridiculous errors and is not critical for the functionality.

**Reply by shvchk** [reply]
@sylirre since that was 4 years ago, maybe it's worth trying again? Considering that xxhash is now default and is much faster ([~13x compared to md5](https://rurban.github.io/smhasher/doc/phone.html) used by Termux rsync by default), which is even more important on low power devices Termux is mostly used on.

Not critical functionality, indeed, though.

**Reply by sylirre** [reply]
I know. Will try it today.

**Reply by sylirre** [reply]
Builds with xxhash support should be available soon.
